packages/flutter/lib/src/rendering/sliver_multi_box_adaptor.dart

```diff
@@ -111,8 +111,15 @@ class SliverMultiBoxAdaptorParentData extends SliverLogicalParentData with Conta
   /// The index of this child according to the [RenderSliverBoxChildManager].
   int index;
 
+  /// Whether to keep the child alive even when it is no longer visible.
+  bool keepAlive = false;
+
+  /// Whether the widget is currently in the
+  /// [RenderSliverMultiBoxAdaptor._keepAliveBucket].
+  bool _keptAlive = false;
+
   @override
-  String toString() => 'index=$index; ${super.toString()}';
+  String toString() => 'index=$index; ${ keepAlive == true ? "keepAlive; " : ""}${super.toString()}';
```

Comment:
Nothing prevents it from being false, and I wouldn't want the toString to crash if it was false (which it would if I didn't do `== true`).